pkgs/by-name/bl/bluesky-pds/package.nix

```
@@ -80,8 +81,9 @@ stdenv.mkDerivation (finalAttrs: {
     runHook postInstall
   '';
 
-  passthru.tests = {
-    inherit (nixosTests) pds;
+  passthru = {
+    tests = { inherit (nixosTests) bluesky-pds; };
```

```suggestion
    tests = lib.optionalAttrs stdenv.hostPlatform.isLinux { inherit (nixosTests) bluesky-pds; };
```

This prevents OfBorg from always attempting to build this on Darwin